clarifai_model_utils/llm_eval/evaluator/harness_eval/doc/new_template.md

```diff
@@ -13,7 +13,7 @@ Suppose you want to use single metric says `f1` and you have your own implementa
   ```yaml
   group:
   - f1
-  task: f1
+  task: f1 # Name of the task
```

Comment:
I would go further and use something obviously different from the name of the function, based on the previous comment on confusion.
```suggestion
  task: calculate_f1 # Name of the task
```